src/backend/distributed/citus.control

```diff
@@ -1,6 +1,6 @@
 # Citus extension
 comment = 'Citus distributed database'
-default_version = '9.3-2'
+default_version = '9.3-3'
```

Comment:
Lately we've been keeping a single file per release and adding to it. That helps somewhat in having many UDF versions for the same release.